### PR TITLE
[17.0][MIG] event_min_seat: Migration to version 17.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT51923

@chienandalu @pedrobaeza please review

### CODE REVIEW

File: event_min_seat/tests/test_event_min_seat.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def setUpClass(cls):
         cls.event = cls.env["event.event"].create(
             {
                 "name": "Test event",
-                "event_type_id": cls.env.ref("event.event_type_data_ticket").id,
+                "event_type_id": cls.env.ref("event.event_type_0").id,
```
Comment (pedrobaeza) [inline event_min_seat/tests/test_event_min_seat.py]: Why using this demo data instead of the type created in the test (or create another one if needed)?

**Review by pedrobaeza — APPROVED** [review]

File: event_min_seat/tests/test_event_min_seat.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply event_min_seat/tests/test_event_min_seat.py]: It was created with demo data, but I make the change to save this maintenance in future migrations.

Comment (pilarvargas-tecnativa) [reply event_min_seat/tests/test_event_min_seat.py]: change done, thanks

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Pull request merged**